<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BTPref</class>
 <widget class="QWidget" name="BTPref">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>477</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="dhtGroupBox">
     <property name="title">
      <string>Features</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QCheckBox" name="kcfg_dhtSupport">
        <property name="toolTip">
         <string>DHT is a trackerless protocol to find peers sharing the same torrents as you do.</string>
        </property>
        <property name="text">
         <string>Use DHT to get additional peers</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>UDP port for DHT communications:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="kcfg_dhtPort">
          <property name="toolTip">
           <string>&lt;p&gt;UDP port to use for the DHT protocol.&lt;/p&gt;
&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Attention:&lt;/span&gt; If you are behind a router, this port needs to be forwarded to accept incoming DHT requests. The UPnP plugin can do this for you.&lt;/p&gt;</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>65535</number>
          </property>
          <property name="value">
           <number>4444</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="kcfg_pexEnabled">
        <property name="toolTip">
         <string>Whether or not to use ÂµTorrent compatible peer exchange.</string>
        </property>
        <property name="text">
         <string>Use peer exchange</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="kcfg_webseedsEnabled">
        <property name="toolTip">
         <string>Enable or disable the use of webseeds when they are present in a torrent.</string>
        </property>
        <property name="text">
         <string>Use webseeds</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="kcfg_checkWhenFinished">
        <property name="toolTip">
         <string>When a torrent has finished downloading, do a full data check on the torrent.</string>
        </property>
        <property name="text">
         <string>Check data when download is finished</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Encryption</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QCheckBox" name="kcfg_useEncryption">
        <property name="toolTip">
         <string>&lt;p&gt;Protocol encryption is useful when your ISP is slowing down bittorrent connections. &lt;/p&gt;
&lt;p&gt;The encryption will prevent your bittorrent traffic to be flagged as bittorrent traffic, and so the ISP will not slow it down.&lt;/p&gt;</string>
        </property>
        <property name="text">
         <string>Use protocol encryption</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="kcfg_allowUnencryptedConnections">
        <property name="toolTip">
         <string>Not all clients support encryption, and some people have encryption disabled. If you want to connect to those peers, you need to have this option enabled.</string>
        </property>
        <property name="text">
         <string>Allow unencrypted connections</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Tracker</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="kcfg_useCustomIP">
        <property name="toolTip">
         <string>Instead of allowing the tracker to determine your IP address, tell the tracker which IP address to use. Use this when you are behind a proxy.</string>
        </property>
        <property name="text">
         <string>Send the tracker a custom IP address or hostname</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Custom IP address or hostname:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="kcfg_customIP">
          <property name="toolTip">
           <string>Custom IP address or hostname to send to the tracker. Hostnames will be resolved at runtime and the resolved IP address will be sent to the tracker.</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>kcfg_dhtSupport</sender>
   <signal>toggled(bool)</signal>
   <receiver>kcfg_dhtPort</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>355</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>370</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kcfg_useEncryption</sender>
   <signal>toggled(bool)</signal>
   <receiver>kcfg_allowUnencryptedConnections</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>215</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kcfg_useCustomIP</sender>
   <signal>toggled(bool)</signal>
   <receiver>kcfg_customIP</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>304</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
